### ISSUE TITLE
[Messaging] conversation.last_message should be nullable

### ISSUE THREAD

**Issue by l2succes** [artsy/metaphysics#665]
<img width="300" src="https://user-images.githubusercontent.com/296775/27338958-c0f9c100-55d6-11e7-9710-b16771e36e17.png" />

**Comment by mzikherman** [comment]
I'm gonna close this since it seems like it was b/c of some simultaneous data migrations around encryption happening.

We'll make new inquiries and can re-open or create new schema issues as needed (from a cleaner slate of fresh inquiries).

**Comment by mzikherman** [comment]
Yup I've been noticing this- in the few seconds or so after placing an inquiry, I guess we can retrieve the conversation but `last_message` is null for a bit.

**Comment by alloy** [comment]
Does it make sense for us to be able to retrieve the conversation while `last_message` is still `null`? Like, can we show anything yet? If not, maybe MP should return `null` for the conversation while `last_message` is still `null`?

**Comment by mzikherman** [comment]
Looking at the code in Impulse/Radiation a bit more closely, it looks like the Conversation is created in Impulse -> Messages created in Radiation -> Webhook back to create messages in Impulse.

So, we have a whole other step in between the time that a conversation is available in Impulse, and when `last_message` is available in Impulse.

**Comment by mzikherman** [comment]
> Does it make sense for us to be able to retrieve the conversation while last_message is still null? Like, can we show anything yet? If not, maybe MP should return null for the conversation while last_message is still null?

Yea I think that's right. If `last_message` is null that means you just initiated the inquiry (and there isn't even any proper messages 'yet') so we should probably just skip those conversations altogether.
